File: apps/institution-api/Models/VwCurrentInstitution.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace ProviderApi.Models
{
    public partial class VwCurrentInstitution
    {
        public int InstitutionId { get; set; }
        public string OperatingName { get; set; }
        public string InstitutionName { get; set; }
        public string InstitutionLegalName { get; set; }
        public string InstitutionShortName { get; set; }
        public string InstitutionType { get; set; }
        public string OperatingType { get; set; }
        public string HomeJurisdiction { get; set; }
        public string InstitutionCategory { get; set; }
        public string InstitutionCode { get; set; }
        public int? ProvincialInstitutionNumber { get; set; }
        public string FederalInstitutionNumber { get; set; }
        public string InstitutionStatus { get; set; }
        public DateTime BusinessStartDate { get; set; }
        public DateTime BusinessEndDate { get; set; }
        public Guid? InstitutionGuid { get; set; }
    }
}

File: libs/common/models/vw-current-institution.ts



export class VwCurrentInstitution {
    public institutionId: number = 0;
    public operatingName: string = '';
    public institutionName: string = '';
    public institutionLegalName: string = '';
    public institutionShortName: string = '';
    public institutionType: string = '';
    public operatingType: string = '';
    public homeJurisdiction: string = '';
    public institutionCategory: string = '';
    public institutionCode: string = '';
    public provincialInstitutionNumber: number;
    public federalInstitutionNumber: string = '';
    public institutionStatus: string = '';
    public businessStartDate: string = '';
    public businessEndDate: string = '';
    public institutionGuid: string;

    constructor(init? : Partial<VwCurrentInstitution>) {
        
        Object.assign(this, init);
    }
}

